src/@types/index.ts

```diff
@@ -1,102 +1,109 @@
 import { BigNumber } from 'ethers'
 
+export enum State {
+  Up = 'UP',
+  Down = 'DOWN',
+  Warning = 'WARNING'
+}
+
 export interface Status {
   network: string
-  aquarius?: AquariusStatus
-  provider?: ProviderStatus
-  subgraph?: SubgraphStatus
-  market?: 'UP' | 'DOWN'
-  port?: 'UP' | 'DOWN'
-  faucet?: FaucetStatus
-  operatorService?: OperatorStatus
-  dataFarming?: 'UP' | 'DOWN'
-  daoGrants?: 'UP' | 'DOWN'
+  currentBlock: number
+  market: State
+  port: State
+  faucet: FaucetStatus | Record<string, never>
+  aquarius: AquariusStatus
+  provider: ProviderStatus
+  subgraph: SubgraphStatus
+  operator: OperatorStatus
+  dataFarming: State
+  daoGrants: State
+  lastUpdatedOn: number
 }
 export interface ProviderStatus {
-  status?: 'UP' | 'DOWN' | 'WARNING'
+  status?: State
   response?: number
   version?: string
   latestRelease?: string
-  block?: number
-  latestBlock?: number
 }
 
 export interface AquariusStatus {
-  status?: 'UP' | 'DOWN' | 'WARNING'
+  status?: State
   response?: number
   chain?: boolean
```

Comment:
why is this boolean?

Reply:
We're requesting the chain list from aquarius and then `chain=true` if the current network is on the list. It wasn't a good name, I'll change the name to `validChainList` so it is clearer what this is

Reply:
hmm, ok, but there should be an array with the actual list of event monitors for each chain and what version they are

Reply:
At the moment `Status` is an array with one `AquariusStatus` for each network so the event monitor version for each chain is there. I could refactor it though so that there is only one `AquariusStatus` and then it has an array of event monitor versions

Reply:
i am a bit confused, aquarius has 2 components, the web service and the event monitor. There is only one web service and an event monitor for each chain. If there is just an array how are you displaying the info regarding the web service. 
the structure should be (like it was described in https://github.com/oceanprotocol/status-service/issues/6) :
```
"aquarius: : {
                 "status": "WARNING",
                 "response": 200,
                 "chains": [{
                     "id": 1,
                     "last_block" :234242
                     "version": "1.3.2"}],
                 "version": "4.2.0",
                 "latestRelease": "4.4.0",
                 "validQuery": 1,
 },
```

Reply:
At the moment it's returning an object for each chain, even for components that are the same across chains (market, port etc) the response structure looks like this:

```JSON
[
    {
        "network": "mainnet",
        "lastUpdatedOn": 1664560056769,
        "currentBlock": 15647738,
        "market": "UP",
        "port": "UP",
        "aquarius": {
            "status": "WARNING",
            "response": 200,
            "version": "4.4.2",
            "monitorVersion": "4.4.2",
            "latestRelease": "4.4.2",
            "block": 15647739,
            "validQuery": true,
            "validChainList": true
        },
        "provider":  { ... },
        "subgraph": { ... },
        "operator":  { ... },
        "faucet": {}
    },
    {
        "network": "polygon",
        "lastUpdatedOn": 1664560075307,
        "currentBlock": 0,
        "market": "UP",
        "port": "UP",
        "aquarius": {
            "status": "WARNING",
            "response": 200,
            "version": "4.4.2",
            "monitorVersion": "4.4.2",
            "latestRelease": "4.4.2",
            "block": 33763166,
            "validQuery": true,
            "validChainList": true
        },
        "provider":  { ... },
        "subgraph":  { ... },
        "operator": { ... },
    {
        "network": "mumbai",
        "lastUpdatedOn": 1664560195231,
        "currentBlock": 0,
        "market": "UP",
        "port": "UP",
        "aquarius": {
            "status": "WARNING",
            "response": 200,
            "version": "4.4.2",
            "monitorVersion": "4.4.2",
            "latestRelease": "4.4.2",
            "block": 28376826,
            "validQuery": true,
            "validChainList": true
        },
        "provider":  { ... },
        "subgraph":  { ... },
        "operator":  { ... },
    {
        "network": "energyweb",
        "lastUpdatedOn": 1664560125184,
        "currentBlock": 20268443,
        "market": "UP",
        "port": "UP",
        "aquarius": {
            "status": "WARNING",
            "response": 200,
            "version": "4.4.2",
            "monitorVersion": "4.4.2",
            "latestRelease": "4.4.2",
            "block": 2676361,
            "validQuery": true,
            "validChainList": true
        },
        "provider":  { ... },
        "subgraph": { ... },
        "operator":  { ... },
        "faucet": {}
    },
    {
        "network": "moonriver",
        "lastUpdatedOn": 1664560107287,
        "currentBlock": 2676360,
        "market": "UP",
        "port": "UP",
        "aquarius":  { ... },
        "provider":  { ... },
        "subgraph":  { ... },
        "operator":  { ... },
        "faucet":  { ... }
    },
    {
        "network": "moonbase",
        "lastUpdatedOn": 1664560212245,
        "currentBlock": 2929831,
        "market": "UP",
        "port": "UP",
        "aquarius": {
            "status": "WARNING",
            "response": 200,
            "version": "4.4.2",
            "monitorVersion": "4.4.2",
            "latestRelease": "4.4.2",
            "block": 2929831,
            "validQuery": true,
            "validChainList": true
        },
        "provider":  { ... },
        "subgraph":  { ... },
        "operator":  { ... },
        "faucet": { ... }
    }
]
```

So to get the event monitor version for mainnet it would be `status[0].aquarius.monitorVersion`

Reply:
I think what I'm trying to say is that if we keep this structure and add in the `chains: [ ... ]` array it will lead to a lot of duplication, as each Aquarius object will now contain information about all the other chains. So if that's the direction then some general refactoring of the response is needed.

Reply:
But this is totally different from the specs, now i'm more confused.